__tests__/__system__/cli/view/spool-file-by-id/cli.view.spool-file-by-id.test.ts

```diff
@@ -73,6 +73,11 @@ describe("view spool-file-by-id command", () => {
         expect(result.status).toEqual(0);
         const stdout = result.stdout.toString();
         expect(result.stdout.toString()).toContain("IEFBR14");
+        iefbr14Content = iefbr14Content.split("\n").map((line: string) => {
+            // Exclude the potential trailing statement number.
+            const JCL_LINE_LENGTH = 72;
+            return line.substring(0, JCL_LINE_LENGTH);
+        }).join("\n");
```

Comment:
I know this may not be used anywhere else, but maybe it's worth thinking to include this as part of the testing utilities 😄 
No change required